simple_node_todo/server.js

```diff
@@ -1,6 +1,6 @@
 const express = require('express');
 const app = express();
-const port = ;
+const port = express() ;
```

Comment:
Here, you have to port, which should be an integer. you can put const port = 3000 instead.